api/resources_portal/views/material_request.py

```diff
@@ -0,0 +1,173 @@
+from django.forms.models import model_to_dict
+from rest_framework import serializers, viewsets
+from rest_framework.permissions import BasePermission, IsAuthenticated
+from rest_framework.response import Response
+
+from resources_portal.models import MaterialRequest, Notification
+from resources_portal.views.material import MaterialSerializer
+from resources_portal.views.relation_serializers import AttachmentRelationSerializer
+from resources_portal.views.user import UserSerializer
+
+SHARER_MODIFIABLE_FIELDS = {"status", "executed_mta_attachment"}
+
+REQUESTER_MODIFIABLE_FIELDS = {"irb_attachment", "requester_signed_mta_attachment"}
+
+
+class MaterialRequestSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = MaterialRequest
+        fields = (
+            "id",
+            "is_active",
+            "status",
+            "assigned_to",
+            "executed_mta_attachment",
+            "irb_attachment",
+            "material",
+            "requester",
+            "requester_signed_mta_attachment",
+            "created_at",
+            "updated_at",
+        )
+        read_only_fields = ("id", "created_at", "updated_at")
+
+
+class MaterialRequestDetailSerializer(MaterialRequestSerializer):
+    assigned_to = UserSerializer()
+    requester = UserSerializer()
+    material = MaterialSerializer()
```

Comment:
Can you actually use the *RelationSerializer for users and materials as well? The non-relation serializers have nested attributes, which we don't want. For example I'd expect a response from this endpoint to have a user object for who it's assigned to, but I don't need to see every org that user belongs to.